dali/operators/generic/one_hot_test.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <tuple>
+#include "dali/operators/generic/one_hot.h"
+#include "dali/kernels/kernel_params.h"
+#include "dali/kernels/test/kernel_test_utils.h"
+#include "utility"
```

Comment:
```suggestion
#include <utility>
```
We agreed some time ago to use `<>` for "system" headers. If you apply this suggestion, you'll need to move this line just beside `#include <tuple>`, otherwise lint will scream